app/Actions/EditAfter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Actions;
+
+use App\InteractsWithLamboConfig;
+
+class EditAfter
+{
+    use InteractsWithLamboConfig;
+
+    public function __invoke()
+    {
+        $this->createOrEditConfigFile("after", $this->afterFileTemplate());
+    }
+
+    private function afterFileTemplate()
+    {
+        return <<<'TEMPLATE'
```

Comment:
Can we make this a stub file that this class loads?

Reply:
Done.